nginx_vts_exporter.go

```diff
@@ -144,7 +150,7 @@ type Exporter struct {
 	serverMetrics, upstreamMetrics, cacheMetrics map[string]*prometheus.GaugeVec
 }
 
-func newServerMetric(metricName string, docString string, labels []string) *prometheus.GaugeVec {
+func newServerMetric(metricName string, docString string, labels []string, namespace string) *prometheus.GaugeVec {
```

Comment:
Yes, I totally agree. But the point here is we do not need pass the namespace as args, just use the global variable `namespace` instead, this should avoid changing too many codes. What do you think? :)